src/js/components/icons/status/Disabled.js

```diff
@@ -1,22 +1,26 @@
 // (C) Copyright 2014-2015 Hewlett-Packard Development Company, L.P.
 
 var React = require('react');
+var IntlMixin = require('../../../mixins/GrommetIntlMixin');
 
 var Disabled = React.createClass({
 
+  mixins: [IntlMixin],
+
   render: function() {
     var className = 'status-icon status-icon-disabled';
     if (this.props.className) {
       className += ' ' + this.props.className;
     }
     return (
-      <svg className={className} viewBox="0 0 24 24" version="1.1">
-        <g className={"status-icon__base"}>
-          <rect x="0" y="0" width="24" height="24" stroke="none"></rect>
+      <svg className={className} viewBox="0 0 24 24" role="img" aria-labelledby="title-icon-disabled" version="1.1">
+        <title id="title-icon-disabled">{this.getGrommetIntlMessage('Disabled')}</title>
+	<g className={"status-icon__base"}>
+          <rect role="presentation" x="0" y="0" width="24" height="24" stroke="none"></rect>
```

Comment:
Based on this site: http://www.sitepoint.com/tips-accessible-svg/
I would expect we'd put the role on the <svg> tag itself, instead of on each drawing element within it.